src/components/Dialog/ProductTransferDialog.vue

```diff
@@ -52,14 +52,15 @@ const handleSubmit = async () => {
 
   const data = {
     productId: product.id,
-    recipientId: allUsers.value.find((user) => user.name == selectedUser).id,
+    recipientId: allUsers.value.find((user) => user.firstName == selectedUser)
```

Comment:
we could easily hit a bug here because we can have 2 users with the same firstName. If we entered the people in our org, we would break it 😆

Reply:
done